.github/workflows/e2e-tests.yml

```diff
@@ -4,34 +4,39 @@ name: E2E Tests
 on: pull_request
 
 jobs:
+  setup-env:
+    name: 'Load ENV Vars'
+    uses: ./.github/workflows/setup-env.yml
+    secrets: inherit
+
   e2e-tests:
     runs-on: ubuntu-latest
+    needs: [setup-env]
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
       
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: "lts/*"
+          node-version: '20'
```

Comment:
As I was running this, I noticed a warning about an old version of node, so I changed this to specify the node version.

Reply:
I'd prefer to keep this on lts. I'd rather have to fix things when lts gets updated, then get way behind by hardcoding a version